trace/TraceConfig.md

```diff
@@ -5,7 +5,8 @@ sampler, maximum events to be kept, etc.
 
 ## TraceParams
 Represents the set of parameters that users can control 
-* Default `Sampler` - used when creating a Span if no specific sampler is given.
+* Default `Sampler` - used when creating a Span if no specific sampler is given. The default sampler
```

Comment:
I worry that we will want to replace this with a rate limited sampler as the default once this is available. For most applications, rate limited seems to be a better default. Maybe we should make a note that we might replace it in future with a rate limit but it will stil not exceed 1/10000 by default.